pkg/ingester/circuitbreaker.go

```diff
@@ -25,6 +25,7 @@ const (
 	circuitBreakerResultError        = "error"
 	circuitBreakerResultOpen         = "circuit_breaker_open"
 	circuitBreakerDefaultPushTimeout = 2 * time.Second
+	circuitBreakerDefaultReadTimeout = 2 * time.Second
```

Comment:
I think 2 seconds for a read is way too aggressive.

Reply:
@colega What should we put here? 10s?

Reply:
I would start with... 1 minute? 30s maybe.

Reply:
I'll set it to 30s.

Reply:
We use 2min timeout on read path internally.